lib/console.c

```diff
@@ -699,9 +699,15 @@ setup_verbosity(VOID)
 	EFI_STATUS efi_status;
 	UINT8 *verbose_check_ptr = NULL;
 	UINTN verbose_check_size;
+	const CHAR16 * const verbose_var_name =
+#ifdef ENABLE_SHIM_DEVEL
```

Comment:
I would personally do this as a #define rather than littering the variable initialization with ifdefs, but it's not wrong or anything.